Example/Pods/UberSignals/UberSignals/UBSignal.h

```diff
@@ -195,6 +195,7 @@ CreateSignalInterface(UBMutableDictionarySignal, NSMutableDictionary *mutableDic
 - (instancetype)initWithProtocol:(Protocol *)protocol NS_DESIGNATED_INITIALIZER;
 
 - (instancetype)init NS_UNAVAILABLE;
++ (instancetype)new NS_UNAVAILABLE;
```

Comment:
This file is part of a separate framework ([UberSignals](https://github.com/uber/signals-ios)) included via CocoaPods, so shouldn't be included in this PR.